decidim-admin/spec/system/admin_manages_officializations_spec.rb

```diff
@@ -131,4 +131,49 @@
       end
     end
   end
+
+  describe "contacting the user" do
+    let!(:user) { create(:user, organization: organization) }
+
+    before do
+      click_link "Officializations"
+    end
+
+    it "redirect to conversation path" do
+      within "tr[data-user-id=\"#{user.id}\"]" do
+        click_link "Contact"
+      end
+      expect(page).to have_current_path "/conversations/new?recipient_id=#{user.id}"
```

Comment:
I think we can say `decidim.new_conversation_path(recipient_id: user.id)` here, for the same reason as before.

Reply:
Now I see how you use it ! I missed the recipient_id part